contrib/sssd.spec.in

```diff
@@ -857,6 +858,8 @@ done
 %dir %{_libdir}/%{name}
 %dir %{_libdir}/%{name}/modules
 %{_libdir}/%{name}/modules/sssd_krb5_localauth_plugin.so
+%{_libdir}/%{name}/modules/sssd_krb5_idp_plugin.so
+%{_datadir}/sssd/krb5-snippets/sssd_enable_idp
```

Comment:
Do we really want to enable plugins "by default" (via package installation) orwould it be better to defer this to ipa-*-install scripts?

Reply:
In order to activate pre-auth method, IPA KDB driver will need to provide information in the Kerberos principal entry, only then KDC side of the pre-auth plugin will 'activate'. So this is safe to enable. On the client side the plugin only triggered if KDC returns its OID in the list of enabled ones.